### ISSUE TITLE
Idea: attach image to a date

### ISSUE THREAD

**Issue by ferreiro** [cheeaun/life#55]
Hello guys! First congrats for the project and idea. I love it!

Today, I had an idea to improve the functionality of the life project. Could be awesome to attach an image to a particular date. This is a really cool way to remember the "important date" but also has a visual memory. 

Regards.
Jorge.

**Comment by hotoo** [comment]
What's it look like?

Life project support attach image in event message.

**Comment by clawoflight** [comment]
@hotoo How do you do that?

**Comment by hotoo** [comment]
```markdown
* 2015/01/01-~ ![img](url)
```

**Comment by clawoflight** [comment]
@hotoo Yeah, that *should* work in markdown. But it doesn't work in life.
All that happens is that the image is linked, but the `!` is not respected and even shown in the text.

I assume that @ferreiro would want the image to be shown in the diagram, which is also what I want.

**Comment by hotoo** [comment]
Is my fault, life is not support image markdown syntax now. life support link syntax.

You can make a PR to life, or use [Markline](https://github.com/hotoo/markline)

**Comment by clawoflight** [comment]
@hotoo markline looks good, but it looks like it won't work on github pages?

**Comment by hotoo** [comment]
```bash
$ markline build life.md
$ ghp-import dist
$ git push origin gh-pages
```
